### DISCUSSION TITLE
What is the status of this project?

### DISCUSSION THREAD

**Discussion by HandsOfVelika** [OldBoyCoder/openPER#55] [Q&A]
Hy! I am Rudolf from Hungary, and I just found this project. What is the status of them? I want a "linux based" type of the fiat eper, and i see it is writen in C# and .NET Is it can works on linux?

Thank you for your answer in advance!

**Comment by OldBoyCoder** [comment]
> Hy! I am Rudolf from Hungary, and I just found this project. What is the status of them? I want a "linux based" type of the fiat eper, and i see it is writen in C# and .NET Is it can works on linux?
> 
> Thank you for your answer in advance!

Hi Rudolf,
I'm sorry that I missed seeing your question. I have never tried to run openPer under Linux but I think it should be possible. If I get some time then I may try it out 
If you just want to use it then a version is hosted at https://eper.fiatforum.com/
